package/rmt-migration.service

```diff
@@ -7,6 +7,7 @@ Before=rmt.service
 [Service]
 Type=oneshot
 User=_rmt
+Environment=RAILS_LOG_TO_JOURNALD=1
```

Comment:
should I rename it to something like `LOG_TO_JOURNALD`? - since CLI does not really init rails

Reply:
Good idea

Reply:
@Sergeykot `LOG_TO_JOURNALD` doesn't work if you consider that you can run these commands in the CLI.

Reply:
Ignore my previous comment